sdk/ai/azure-ai-agents/azure/ai/agents/operations/_patch.py

```diff
@@ -0,0 +1,2234 @@
+
+            logger.debug("Current run ID: %s with status: %s", run.id, run.status)
+
+        return run
+
+    @overload
+    def stream(
+        self,
+        thread_id: str,
+        *,
+        include: Optional[List[Union[str, _models.RunAdditionalFieldList]]] = None,
+        agent_id: str,
+        content_type: str = "application/json",
+        model: Optional[str] = None,
+        instructions: Optional[str] = None,
+        additional_instructions: Optional[str] = None,
+        additional_messages: Optional[List[_models.ThreadMessageOptions]] = None,
+        tools: Optional[List[_models.ToolDefinition]] = None,
+        temperature: Optional[float] = None,
+        top_p: Optional[float] = None,
+        max_prompt_tokens: Optional[int] = None,
+        max_completion_tokens: Optional[int] = None,
+        truncation_strategy: Optional[_models.TruncationObject] = None,
+        tool_choice: Optional["_types.AgentsToolChoiceOption"] = None,
+        response_format: Optional["_types.AgentsResponseFormatOption"] = None,
+        parallel_tool_calls: Optional[bool] = None,
+        metadata: Optional[Dict[str, str]] = None,
+        event_handler: None = None,
+        **kwargs: Any,
+    ) -> _models.AgentRunStream[_models.AgentEventHandler]:
+        """Creates a new stream for an agent thread.
+
+        :param thread_id: Required.
+        :type thread_id: str
+        :keyword include: A list of additional fields to include in the response.
+         Currently the only supported value is
+         ``step_details.tool_calls[*].file_search.results[*].content`` to fetch the file search result
+         content. Default value is None.
+        :paramtype include: list[str or ~azure.ai.agents.models.RunAdditionalFieldList]
+        :keyword agent_id: The ID of the agent that should run the thread. Required.
+        :paramtype agent_id: str
+        :keyword content_type: Body Parameter content-type. Content type parameter for JSON body.
+         Default value is "application/json".
+        :paramtype content_type: str
+        :keyword model: The overridden model name that the agent should use to run the thread. Default
+         value is None.
+        :paramtype model: str
+        :keyword instructions: The overridden system instructions that the agent should use to run the
+         thread. Default value is None.
+        :paramtype instructions: str
+        :keyword additional_instructions: Additional instructions to append at the end of the
+         instructions for the run. This is useful for modifying the behavior
+         on a per-run basis without overriding other instructions. Default value is None.
+        :paramtype additional_instructions: str
+        :keyword additional_messages: Adds additional messages to the thread before creating the run.
+         Default value is None.
+        :paramtype additional_messages: list[~azure.ai.agents.models.ThreadMessage]
```

Comment:
Should be.... in multiple spots

```suggestion
        :paramtype additional_messages: list[~azure.ai.agents.models.ThreadMessageOptions]
```

Reply:
Did a global replacement

Reply:
Done